docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -1,21 +1,21 @@
 == AeroGear Digger Installation
-https://github.com/aerogear/aerogear-digger-installer[Digger Installer^] - GitHub repo with Ansible playbook used for Digger installation.
+The AeroGear Digger OpenShift project allows you to setup a Jenkins CI/CD server for building mobile applications using OpenShift containers.
```

Comment:
I think it would be better to say `The AeroGear Digger project allows you to setup a Jenkins CI/CD server on OpenShift for building mobile applications using containers.

Reply:
oh yeah, it's not only an OpenShift project

resolved